srcpkgs/discover/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'discover'
+pkgname=discover
+version=5.26.4
+revision=1
+archs="x86_64"
+#wrksrc=
+#create_wrksrc=yes
+build_style=cmake
+#configure_args=""
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends="pkg-config extra-cmake-modules"
+makedepends="plasma-workspace-devel AppStream-devel AppStream-qt PackageKit-devel qt5-devel qt5-webview-devel gettext-devel purpose-devel"
+depends="PackageKit AppStream AppStream-qt qt5 qt5-webview gettext purpose PackageKit-qt"
```

Comment:
most/all should be autodetected by xbps-src

Reply:
Do I just remove all of it?

Reply:
probably, yes